src/index.ts

```diff
@@ -11,15 +11,18 @@ const dir = __dirname.endsWith('dist/src') ? '../' : '';
 const schemaFile = path.join(__dirname, `${dir}../src/schema.gql`);
 const schema = fs.readFileSync(schemaFile, 'utf8');
 
+const sepoliaConfig = createConfig('sepolia');
 const baseConfig = createConfig('base');
 
+const sepoliaIndexer = new evm.EvmIndexer(createEvmWriters('sepolia'));
 const baseIndexer = new evm.EvmIndexer(createEvmWriters('base'));
 
 const checkpoint = new Checkpoint(schema, {
   logLevel: LogLevel.Info,
   prettifyLogs: true
 });
 
+checkpoint.addIndexer('sepolia', sepoliaConfig, sepoliaIndexer);
```

Comment:
We are missing Ethereum mainnet, and the server should be indexer either the mainnets or the testnets not both. 

Also we should use the EIP-3770 network id as indexer name, not the full name.
```suggestion
checkpoint.addIndexer('sep', sepoliaConfig, sepoliaIndexer);
```

Reply:
Schnaps was not deployed on mainnet. Should we deploy on mainnet? Are there other chains you would like us to deploy on?

Reply:
Yes we should deploy on mainnet, and thats it. Base, Sepolia and Mainnet

Reply:
e87f27145bec381d8102012577852950762cf12e